airgun/views/task.py

```diff
@@ -95,3 +97,15 @@ class task(SatTab):
         progressbar = ProgressBar()
         output = TaskReadOnlyEntry(name='Output')
         errors = TaskReadOnlyEntry(name='Errors')
+
+    def wait_for_result(self, timeout=60, delay=1):
+        """Wait for invocation job to finish"""
+        wait_for(
+            lambda: (self.is_displayed and self.task.progressbar.is_displayed
+                     and self.task.result.read() == 'success'
+                     and ('0 fail' in self.task.output.read())
```

Comment:
I don't know do we really need these checks?

Reply:
No we do not need it. :+1: